openedx/core/djangoapps/enrollments/errors.py

```diff
@@ -52,3 +52,8 @@ class EnrollmentApiLoadError(CourseEnrollmentError):
 class InvalidEnrollmentAttribute(CourseEnrollmentError):
     """Enrollment Attributes could not be validated"""
     pass  # lint-amnesty, pylint: disable=unnecessary-pass
+
+
+class EnrollmentNotUpdatableError(CourseEnrollmentError):
+    """The requested enrollment could not be updated."""
+    pass  # lint-amnesty, pylint: disable=unnecessary-pass
```

Comment:
you don't need pass here - having a docstring is actually enough to make a valid class definition.